internal/proxy/impl.go

```diff
@@ -3080,15 +3068,20 @@ func (node *Proxy) CalcDistance(ctx context.Context, request *milvuspb.CalcDista
 			queryShardPolicy:   roundRobinPolicy,
 		}
 
+		items := make([]zapcore.Field, 0)
+		items = append(items, zap.String("traceID", traceID))
+		items = append(items, zap.String("role", typeutil.ProxyRole))
+		items = append(items, zap.Int64("msgID", qt.Base.MsgID))
+		items = append(items, zap.Uint64("timestamp", qt.Base.Timestamp))
```

Comment:
I don't think we want all of them logged, for example we definitely don't need "traceID", "role", "msgID" and "db"